Cargo.toml

```diff
@@ -25,27 +25,27 @@ tag-prefix = ""
 [dependencies]
 base64 = "0.11"
 error-chain = { version = "0.12", default-features = false }
-futures = "0.1"
-http = "0.1"
+futures = "0.3"
+http = "0.2"
 libflate = "0.1"
 log = "0.4"
 mime = "0.3"
 mockito = { version = "0.22", optional = true }
 regex = "^1.1.0"
-serde = "1"
-serde_derive = "1"
+serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 strum = "0.16"
 strum_macros = "0.16"
 tar = "0.4"
-tokio = "0.1"
+tokio = "0.2"
```

Comment:
Should I bump the minor version as well or do you update it?

Reply:
I'm not sure I get your question, but feel free to keep the bump in this PR.

Reply:
Sorry, the question is unclear. 

I wanted to ask, if I should bump the version of `dkregistry` itself. For example: 

```toml
[package]
name = "dkregistry"
version = "0.4.0-alpha.0"
authors = ["Luca Bruno <lucab@debian.org>", "Stefan Junker <sjunker@redhat.com>"]
license = "MIT/Apache-2.0"
```

Reply:
Ah, yes please. `0.4.0-alpha.0` would be perfect.